pkg/imgpkg/cmd/push_test.go

```diff
@@ -218,15 +255,19 @@ func Cleanup(dirs ...string) {
 }
 
 func createBundleDir(loc, imagesYaml string) error {
-	if imagesYaml == "" {
-		imagesYaml = emptyImagesYaml
-	}
-
-	bundleDir := filepath.Join(loc, ".imgpkg")
+	bundleDir := filepath.Join(loc, bundle.ImgpkgDir)
 	err := os.Mkdir(bundleDir, 0700)
 	if err != nil {
 		return err
 	}
 
+	if imagesYaml == "" {
+		imagesYaml = emptyImagesYaml
+	}
 	return ioutil.WriteFile(filepath.Join(bundleDir, "images.yml"), []byte(imagesYaml), 0600)
 }
+
+func createEmptyBundleDir(loc string) error {
+	bundleDir := filepath.Join(loc, bundle.ImgpkgDir)
```

Comment:
```suggestion
	bundleDir := filepath.Join(loc, ".imgpkg")
```
Same as the above comment

Reply:
Maybe we can call this function from the `createBundleDir` function

Reply:
Hm. Good thinking. But then... the `createBundleDir` would also have to return the `bundleDir` needed in LN 265. Because  `createBundleDir` is called in the context of a test that doesn't need this value, I think it's best not to do this refactoring.

Reply:
You are right, let's keep it as is